### PR TITLE
[ru] Country names translated

### PR DESCRIPTION
**Pull request by andrey-helldar**

### CODE REVIEW

File: json/ru.json [hunk 1]
```diff
@@ -201,42 +201,42 @@
     "Forgot Your Password?": "Забыли пароль?",
     "Forgot your password?": "Забыли пароль?",
     "Forgot your password? No problem. Just let us know your email address and we will email you a password reset link that will allow you to choose a new one.": "Забыли пароль? Нет проблем. Просто сообщите Ваш email-адрес и мы пришлём Вам ссылку для сброса пароля.",
-    "France": "France",
-    "French Guiana": "French Guiana",
-    "French Polynesia": "French Polynesia",
-    "French Southern Territories": "French Southern Territories",
+    "France": "Франция",
```
Comment (caouecs) [inline json/ru.json:204]: Interesting ;)

**Review by caouecs — APPROVED** [review]
nothing :)

**Comment by caouecs** [comment]
Спасибо

File: json/ru.json (hunk 1 above)
Comment (andrey-helldar) [reply json/ru.json:204]: Yes, in Russian, "France" is spelled "Франция" and pronounced like "Franciya" :)

**Pull request merged**